LAD-AMA-Common/telegraf_utils/telegraf_config_handler.py

```diff
@@ -750,6 +750,13 @@ def handle_config(config_data, me_url, mdsd_url, is_lad):
     if "virtualMachineScaleSets" in az_resource_id: 
         az_resource_id = "/".join(az_resource_id.split("/")[:-2])
 
+    virtual_machine_name = ""
```

Comment:
Can you add similar to previous section comment that this is needed specifically for ME ? and what ME expect the id to be in case of flexible after parsing? Example of resource id after parsing that ME can handle